charts/kserve-resources/templates/clusterrole.yaml

```diff
@@ -12,50 +11,12 @@ rules:
   resources:
   - subjectaccessreviews
   verbs: ["create"]
-
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: kserve-manager-role
 rules:
-- apiGroups:
-  - admissionregistration.k8s.io
```

Comment:
are we removing t he admission webhook roles ?

Reply:
No, It is rearranged https://github.com/kserve/kserve/pull/3970/files#diff-476b4c5d97f5649cb09860e052989f34d1e2c464ba6efd27a98b482cb0741f1eR60